### DISCUSSION TITLE
Mix fan curve Max function issue

### DISCUSSION THREAD

**Discussion by alex3xelax** [Rem0o/FanControl.Releases#681] [Q&A] [answered]
![fan_control_mix](https://user-images.githubusercontent.com/88495460/165276293-9925f453-6b4c-4f6a-8b91-40db81f56dc2.png)

hi
As you  can see in the screenshoot, the "**max function**" not select the **cpu->case** graph with highter temp also if the cpu temp is highter

any idea ?
thanks

**Comment by xhul-dev** [comment, marked as answer]
Hello there.
The "Max" function takes the highest %, it doesn't directly care about temperature.
And since both graph curves are at 30%, 30% is correctly the highest.
What's between parentheses is irrelevant in that case, because both % are equal.
